src/ape/api/compiler.py

```diff
@@ -47,7 +48,7 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
             contract_filepaths (List[pathlib.Path]): A list of source file paths to compile.
 
         Returns:
-            List[:class:`~ape.type.contract.ContractType`]
+            List[:class:`ethpm_types.contract_type.ContractType`]
```

Comment:
I don't think this works - linking to another project's docs in method docs. Correct me if I am wrong.
If it does not work, we can just do a normal back-tick thing.